forge-ai/src/main/java/forge/ai/ComputerUtil.java

```diff
@@ -433,12 +427,41 @@ public static Card getCardPreference(final Player ai, final Card activate, final
                 final CardCollection sacMeList = CardLists.filter(typeList, new Predicate<Card>() {
                     @Override
                     public boolean apply(final Card c) {
-                        return c.hasSVar("SacMe") && Integer.parseInt(c.getSVar("SacMe")) == priority;
+                        return (c.hasSVar("SacMe") && Integer.parseInt(c.getSVar("SacMe")) == priority)
+                                || (priority == 1 && shouldSacrificeThreatenedCard(ai, c, sa));
```

Comment:
sa.getAPI() NPE on 
```
shouldSacrificeThreatenedCard(ai, c, sa)
```
because of 
```
getCardPreference(ai, activate, pref, typeList, null)
```
 line above... hence NPE is thrown.. 
 
![image](https://github.com/Card-Forge/forge/assets/9781539/e317f5c0-5774-41f5-a22c-f03e0574ff1f)

Reply:
Thanks! :) Fixed.